exercises/linked-list/linked-list.t

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env perl6
+
+use Test;
+use JSON::Tiny;
+
+use lib ( my $dir = IO::Path.new($?FILE).parent ).path;
+
+my $module_name = %*ENV<EXERCISM>.so ?? 'Example' !! 'LinkedList';
+my @potential_module = <p6 pm6 pm>.map:  $module_name ~ '.' ~ *; 
```

Comment:
I've noticed a few of your tests have this. Do you know you it's possible to interpolate a string into a package name using `::($module)`?

There's an example here: https://docs.perl6.org/syntax/require#language_documentation_Modules

It's fine as it is, but I just wanted to check as it may save a few lines 🙂